girder/utility/webroot.py

```diff
@@ -88,9 +90,18 @@ def __init__(self, templatePath=None):
             'plugins': [],
             'apiRoot': '',
             'staticRoot': '',
-            'title': 'Girder'
+            'title': 'Girder',
+            'contactEmail': ModelImporter.model('setting').get(SettingKey.CONTACT_EMAIL_ADDRESS)
         }
 
+        events.bind('model.setting.save.after', CoreEventHandler.WEBROOT_SETTING_CHANGE,
+                    self._onSettingSave)
+
+    def _onSettingSave(self, event):
```

Comment:
This sort of thing (which exists in other places in Girder thanks to yours truly) will not  work in a distributed installation. 

For the purposes of this PR this might be fine, but I think that #2274 might try to remove these patterns. For more detail about it there's a thread on girder-developers with the subject "Signalling between multiple Girder servers".